setup.py

```diff
@@ -53,6 +53,9 @@
 with open(PATH) as filehandle:
     VERSION = json.load(filehandle)["version_string"]
 
+with open(os.path.join(os.path.abspath(os.path.dirname(__file__)), "requirements.in")) as f:
```

Comment:
Won't this break if `requirements.in` isn't part of the package?

I could see us either:
a) checking whether the file exists, and only running this block if it does, and otherwise falling back to `req`, or
b) including `requirements.in` in the package and always running this block. I think this means we can get rid of `reqs` above.

Reply:
Sorry, you're absolutely right. It looks like I rushed this a bit.